generator/gen_daal4py.py

```diff
@@ -942,22 +942,28 @@ def gen_daal4py(daalroot, outdir, version, warn_all=False, no_dist=False, no_str
         f.write(cpp_h)
     with open(jp(outdir, 'daal4py_cpp.cpp'), 'w') as f:
         f.write(cpp_cpp)
-
     with open(jp('src', 'gettree.pyx'), 'r') as f:
         pyx_gettree = f.read()
 
-    pyx_modelbuilder = ''
+    pyx_gbt_model_builder = ''
+    pyx_log_reg_model_builder = ''
     if 'algorithms::gbt::classification' in iface.namespace_dict and \
        'ModelBuilder' in iface.namespace_dict['algorithms::gbt::classification'].classes or \
        'algorithms::gbt::regression' in iface.namespace_dict and \
        'ModelBuilder' in iface.namespace_dict['algorithms::gbt::regression'].classes:
-        with open(jp('src', 'modelbuilder.pyx'), 'r') as f:
-            pyx_modelbuilder = f.read()
+        with open(jp('src', 'gbt_model_builder.pyx'), 'r') as f:
+            pyx_gbt_model_builder = f.read()   
+    if iface.version[0] >= 2021 and iface.version[1] >= 2 and \
```

Comment:
For gold version. Check please

```suggestion
    if iface.version[0] >= 2021 and iface.version[1] >= 1 and \
```

Reply:
Corrected